### PR TITLE
Change localhost to 127.0.0.1 in wazuh.yml

### LINKED ISSUE #7277: Wazuh dashboard `localhost` references
Following https://github.com/wazuh/wazuh-documentation/issues/7225, the `localhost` references are not accurate any longer in v4.8.0. We need to review the `Wazuh dashboard` section for these occurrences.

- https://documentation.wazuh.com/current/user-manual/wazuh-dashboard/index.html

For example:

image

### PR DESCRIPTION
**Pull request by asteriscos**
# Description

This pull request changes `localhost` to `127.0.0.1` in **_wazuh-dashboard_** and **_user-administration_** sections.

Closes #7277 and #7276

### CODE REVIEW

**Review by yenienserrano — DISMISSED** [review]
LGTM

File: source/user-manual/user-administration/ldap.rst
Comment (javimed) [inline source/user-manual/user-administration/ldap.rst:1]: What about other occurrences of `localhost`? For example those found in:

- https://github.com/wazuh/wazuh-documentation/blob/7277-replace-wazuh.yml-localhost-to-127.0.0.1/source/user-manual/user-administration/ldap.rst?plain=1#L154-L163

Are we keeping these commands unmodified?

Comment (asteriscos) [reply source/user-manual/user-administration/ldap.rst:1]: I reviewed the default indexer host specied and changed it to 127.0.01

**Review by Tostti — APPROVED** [review]
LGTM

**Pull request merged**